RRT_star.py

```diff
@@ -37,119 +43,149 @@ def connection(self, va, vb):
         """Return the cost of connecting two verices"""
         return np.linalg.norm(va -vb)
 
-    def expand_graph(self, min_edge_len, max_edge_len, niu=0.8):
-        """Add a vertex to the RRT graph"""
-        q = self.workspace.sample()
-        v_nearest, dist = self.closest_vertex(q)
-        q_nearest = v_nearest.coord
+    
+    def query(
+        self,
+        start,
+        goal,
+        n=1000,
+        min_edge_len=0.5,
+        max_edge_len=1,
+        niu = 0.8
```

Comment:
What is the purpose of the niu variable? Can you add a comment and/or change the name to be more descriptive?

Reply:
niu is the maximum distance that we are willing to shift the sampled point toward the closest point in the tree. it's a constant that in research papers is usually denoted as niu. I don't know much about it but I can definitely research more